src/tests/dates.test.ts

```diff
@@ -21,6 +28,79 @@ describe('abbreviationForWeekday()', () => {
   });
 });
 
+describe('getDateDiff()', () => {
+  let now;
+  let testDate;
+
+  beforeEach(() => {
+    now = new Date();
```

Comment:
Can this simply be a constant in the module scope? If I understand your tests correctly, this doesn't really need to be updated this frequently.